src/ImageSharp.Web.Providers.AWS/Caching/AWSS3StorageCache.cs

```diff
@@ -17,7 +16,7 @@ namespace SixLabors.ImageSharp.Web.Caching.AWS;
 public class AWSS3StorageCache : IImageCache
 {
     private readonly IAmazonS3 amazonS3Client;
-    private readonly string bucket;
+    private readonly string? bucket;
```

Comment:
We should do a guard check for the bucket name and make this non-nullable.

Reply:
It is non nullable. But to achieve this we have the ugly hack with = null! in the options :(